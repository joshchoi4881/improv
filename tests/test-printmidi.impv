func int main() {
  note[] arr;
  arr = [<1, "wh">, <2, "wh">, <3, "wh">];
  render(arr, "test.mid", 1, 120);
  printmidi("test.mid");
  return 0;
}
