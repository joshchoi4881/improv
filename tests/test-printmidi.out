finished creating test.mid!
# Track 0
	ff 51 03 07 a1 20 
	c0 1a 
	ff 58 04 04 02 18 08 
	90 43 40 
	80 43 00 
	90 40 40 
	80 40 00 
	90 3e 40 
	80 3e 00 
	ff 2f 00 
